app/oauth/index.js

```diff
@@ -38,6 +38,7 @@ class Status extends Component {
         .then(res => res.json().then(json => {
             if (res.status >= 200 && res.status < 400) {
                 this.setState({status: 'success'});
+                setTimeout(() => window.close(), 2000);
```

Comment:
If we're going to close the window after success, then we should remove the success content below since users won't have the time read it.

So, content can just be `content = (<h4>Authorized!</h4>)`

Since we're now authenticating users on app-start, they won't get sent to this page when scheduling a query (and therefore we can lighten up the content)